core/templates/domain/topic/subtopic.model.ts

```diff
@@ -197,9 +197,9 @@ export class Subtopic {
       id: subtopicId,
       title: title,
       skill_ids: [],
-      thumbnail_filename: null,
-      thumbnail_bg_color: null,
-      url_fragment: null
+      thumbnail_filename: '',
+      thumbnail_bg_color: '',
+      url_fragment: ''
```

Comment:
ditto, not a good replacement for `null`

Reply:
Done